GVFS/GVFS.Platform.POSIX/POSIXFileSystem.Shared.cs

```diff
@@ -1,13 +1,13 @@
-﻿namespace GVFS.Platform.Mac
-{
-    public partial class MacFileSystem
-    {
-        public static bool TryGetNormalizedPathImplementation(string path, out string normalizedPath, out string errorMessage)
-        {
-            // TODO(Mac): Properly determine normalized paths (e.g. across links)
-            errorMessage = null;
-            normalizedPath = path;
-            return true;
-        }
-    }
-}
+namespace GVFS.Platform.POSIX
+{
+    public partial class POSIXFileSystem
+    {
+        public static bool TryGetNormalizedPathImplementation(string path, out string normalizedPath, out string errorMessage)
+        {
+            // TODO(POSIX): Properly determine normalized paths (e.g. across links)
```

Comment:
Do you expect that all of the TODO(POSIX) will apply to the Mac platform?

We've been using "TODO(Mac)" as a key when searching for TODOs in the code that we still have for Mac.  I'm wondering if perhaps instead of updating these to TODO(POSIX) we should use TODO(Mac), TODO(Linux)?

@jrbriggs any thoughts/preferences here?

Reply:
I can see pros and cons for each. TODO(Mac), TODO(Linux) would let us knock one off at a time and perhaps leave the other as a reminder to test the implementation on that platform. It would be a bit more verbose. I'm supportive of either if removing TODO(POSIX) means the implementation has been proven correct on both platforms.

Reply:
That sounds good to me, as long as were good about TODO(POSIX) meaning "needs to be done on Mac and Linux" I think we can stick with TODO(POSIX).